src/appleseed.studio/mainwindow/project/projectbuilder.h

```diff
@@ -164,15 +165,11 @@ Entity* ProjectBuilder::edit_entity(
     return new_entity_ptr;
 }
 
-template <>
-inline renderer::Camera* ProjectBuilder::edit_entity(
-    renderer::Camera*                   old_entity,
-    renderer::Scene&                    parent,
-    const foundation::Dictionary&       values) const
+template <typename OldEntity, typename NewEntity>
+inline void copy_transform_sequence(
+    const OldEntity&                   old_entity,
```

Comment:
In the strict context of this function, I think `source_entity` and `destination_entity` would make more sense.